### ISSUE TITLE
[ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './optimizer' is not defined by "exports"

### ISSUE THREAD

**Issue by gdorsi** [BuilderIO/qwik#1737]
### Qwik Version

0.11.0

### Operating System (or Browser)

Linux

### Node Version (if applicable)

v16.13.1

### Which component is affected?

Qwik Rollup / Vite plugin

### Expected Behaviour

`npm run dev` and the other build commands should work.

### Actual Behaviour

`npm run dev` returns this error:

```
❯ npm run dev

> dev
> vite --mode ssr

failed to load config from /home/user/qwik-app/vite.config.ts
error when starting dev server:
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './optimizer' is not defined by "exports" in /home/user/qwik-app/node_modules/@builder.io/qwik/package.json
    at new NodeError (node:internal/errors:371:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:440:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:645:7)
    at resolveExports (node:internal/modules/cjs/loader:482:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/user/qwik-app/vite.config.ts:33:24)
```

### Additional Information

Steps to reproduce:
- Create a new project with `npm create qwik@latest` 
- run `npm run dev`
 
now you should see the error.
 
Checking the qwik package.json this is what I have in the exports field for the optimizer entry:
```
    "./optimizer.cjs": "./optimizer.cjs",
    "./optimizer.mjs": "./optimizer.mjs",
    "./optimizer": {
      "import": "./optimizer.mjs"
    },
```

the require entry is missing.

Changing the optimizer import in `vite.config.ts` into `@builder.io/qwik/optimizer.cjs` fixes the error.

**Comment by ahhshm** [comment]
@gdorsi I'm also running into the same error. Can you please share the code that fixed the problem for you?

**Comment by manucorporat** [comment]
Fixed in 0.11.1

thanks for the issue report
